src/editorManager.js

```diff
@@ -142,7 +142,11 @@ class EditorManager {
 
     if (editorClass) {
       this.activeEditor = getEditorInstance(editorClass, this.instance);
+      this.activeEditor.row = row; // pre-preparation needed by getEditedCell
```

Comment:
Maybe we can use `this.instance.getCell` with `topMost` flag to get `TD` element?

Reply:
Done in 9c0f490d2a0b89473ac5576a11725dd71f8499cc